vaadin-form-layout-flow-parent/vaadin-form-layout-flow/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java

```diff
@@ -279,6 +279,64 @@ public void remove(Component... components) {
         }
     }
 
+    /**
+     * <p>
+     * Server-side component for the {@code <vaadin-form-row>} element. Used to
+     * arrange fields into rows inside a {@link FormLayout} when
+     * {@code #setAutoResponsive(boolean)} is enabled.
+     * </p>
+     * <p>
+     * Each {@code <vaadin-form-row>} always starts on a new row. Fields that
+     * exceed the available columns wrap to a new row, which then remains
+     * reserved exclusively for the fields of that {@code <vaadin-form-row>}
+     * element. wrap components for display in a {@link FormLayout}.
+     * </p>
+     *
+     * @author Vaadin Ltd
+     */
+    @Tag("vaadin-form-row")
+    @NpmPackage(value = "@vaadin/polymer-legacy-adapter", version = "24.7.0-beta1")
+    @JsModule("@vaadin/polymer-legacy-adapter/style-modules.js")
+    @NpmPackage(value = "@vaadin/form-layout", version = "24.7.0-beta1")
+    @JsModule("@vaadin/form-layout/src/vaadin-form-row.js")
+    public static class FormRow extends Component
```

Comment:
Food for thoughts: How about using `Composite` instead of `Component`? 

In the past.. people often used APIs that were never intended for a flow-component to be used (e.g. Tab(s) and add()) so instead of providing all possible methods from the Component class - how about using Composite so that you can publish only those methods you guys really wanna support on newly added "components"?

Reply:
That's a good question. I'll see if it makes sense for us in this case, though. `FormRow` is basically a wrapper component that we use to ensure that the first child element is "forced" into a new row. But there could be some API we do not intend to support, so it's worth checking.

Reply:
While I agree with the point in general, I'm not sure if FormRow would benefit from this. Since it's mostly just a wrapper with no specific content requirements, as Diego mentioned, it seems natural for it to be a Component that can hold anything.